src/index.ts

```diff
@@ -0,0 +1,38 @@
+import { isEnable, defaultOptions } from './config'
+import express from 'express'
+import path from 'path'
+import ExpressoHttpInterceptor from './interceptors/http'
+import html from './views/html'
+import { EXPRESSO_STATIC_PATH, EXPRESSO_PUBLIC_PATH } from './constants'
+
+let httpInterceptor: any
+
+const middleware = (options: any = defaultOptions): any => {
```

Comment:
too much any, we surely know object typing here, at least for options, req, res, next